addons/account/models/chart_template.py

```diff
@@ -149,8 +149,8 @@ def try_loading(self, template_code, company, install_demo=True):
 
         template_code = template_code or company and self._guess_chart_template(company.country_id)
 
-        if template_code == 'syscohada' and template_code != company.chart_template:
-            raise UserError(_("The Syscohada chart template shouldn't be selected directly. Instead, you should directly select the chart template related to your country."))
+        if template_code in ('syscohada', 'syscebnl') and template_code != company.chart_template:
```

Comment:
Ah small nitpicking but i will already approve the PR for the next reviewer, use a set instead of a tuple, it's faster in term of perf :eyes: 
```
template_code in {'syscohada', 'syscebnl'}
```